dalston/style.css

```diff
@@ -4339,7 +4339,6 @@ a {
 		display: flex;
 		align-items: flex-start;
 		justify-content: end;
-		white-space: nowrap;
```

Comment:
I believe we need this to avoid the buttons from stacking on top of each other if the site title is very long